src/Renci.SshNet.IntegrationTests/Common/AsyncSocketListener.cs

```diff
@@ -0,0 +1,393 @@
+﻿using System.Net;
+using System.Net.Sockets;
+
+namespace Renci.SshNet.IntegrationTests.Common
+{
+    public class AsyncSocketListener : IDisposable
```

Comment:
This is a copy of `AsyncSocketListener` from unit tests. We can share this class